index.html

```diff
@@ -255,8 +255,11 @@ <h2>贊助夥伴</h2>
 					<!-- social -->
 					<div class="col-md-4 col-md-push-8">
 						<ul class="footer-social">
-							<li><a href="https://cloudnaitve.tw" class="facebook"><i class="fa fa-facebook"></i></a></li>
-							<li><a href="https://www.youtube.com/channel/UCoYY8K9fbfDtTY7m68UCATA" class="youtube"><i class="fa fa-youtube"></i></a></li>
+                            <li><a href="https://cloudnaitve.tw" class="facebook"><i class="fa fa-facebook"></i></a></li>
+                            <li><a href="https://www.youtube.com/channel/UCoYY8K9fbfDtTY7m68UCATA" class="youtube"><i class="fa fa-youtube"></i></a></li>
+                            <li><a href="https://www.meetup.com/CloudNative-Taiwan/" class="meetup"><i class="fa fa-meetup"></i></a></li>
+                            <li><a href="https://t.me/cntug" class="telegram"><i class="fa fa-telegram"></i></a></li>
+                            <li><a href="https://github.com/cloud-native-taiwan" class="github"><i class="fa fa-github"></i></a></li>
```

Comment:
I'm not a HTML expert, but shouldn't here be an indention?

Reply:
There seems to be no difference between tab and blank in HTML, the only difference is that the code style is inconsistent. So I still adjusted it back, thank you for your suggestion.